bootstrap.ml

```diff
@@ -193,7 +193,7 @@ let ocamldep = get_prog bin_dir "ocamldep"
 let run_ocamllex src =
   let dst = String.sub src ~pos:0 ~len:(String.length src - 1) in
   let x = Sys.file_exists dst in
-  let n = exec "%s -q %s" ocamllex src in
+  let n = exec "\"%s\" -q %s" ocamllex src in
```

Comment:
Shouldn't we use `Filename.quote` here? /cc @dra27 for the windows side of things

Reply:
I’m on Windows, that’s how I found the bug. I think the best patch would be to use Filename.quote_command since the command has arguments.

Reply:
Ok. The Windows CI is failing however :/